src/FixedDataTable.js

```diff
@@ -742,6 +742,8 @@ var FixedDataTable = createReactClass({
           scrollableColumns={state.footScrollableColumns}
           scrollLeft={state.scrollX}
           showScrollbarY={showScrollbarY}
+          isHeaderOrFooter={true}
+          ariaIndex={this.props.rowsCount+2}
```

Comment:
Is the +2 accurate if there's a group header?  Same for other ariaIndex fields

Reply:
I did not test with a group header, will adjust my code.